docs/platforms/javascript/guides/nuxt/manual-setup.mdx

```diff
@@ -82,8 +82,8 @@ Add a `sentry.client.config.ts` file to the root of your project (this is probab
 import * as Sentry from '@sentry/nuxt';
 
 Sentry.init({
-  // If set up, you can use your runtime config here
-  // dsn: useRuntimeConfig().public.sentry.dsn,
+  // If set up, you can use the Nuxt runtime config here
+  // dsn: useRuntimeConfig().public.sentry.dsn, // modify, depending on your custom runtime config
```

Comment:
l: should the new `//` not be in its own line?

Reply:
I added it in this line so people can uncomment this line at it does not produce a syntax error

Reply:
ah, right I see what the intent is.